projects/plugins/boost/app/assets/src/js/features/minify-meta/minify-meta.tsx

```diff
@@ -69,14 +93,24 @@ const MetaComponent = ( { buttonText, placeholder, datasyncKey }: Props ) => {
 							}
 						} }
 					/>
-					<span className={ styles.help }>
+					<div className={ styles.description }>
 						{ __( 'Use a comma (,) to separate the strings.', 'jetpack-boost' ) }
-					</span>
-					<div className={ styles[ 'buttons-container' ] }>
-						<button disabled={ values.join( ', ' ) === inputValue } onClick={ save }>
-							{ __( 'Save', 'jetpack-boost' ) }
-						</button>
 					</div>
+					<Button
+						disabled={ values.join( ', ' ) === inputValue }
+						className={ styles.button }
+						onClick={ save }
+					>
+						{ __( 'Save', 'jetpack-boost' ) }
+					</Button>
+					<Button
+						disabled={ inputValue === defaultValue }
+						onClick={ loadDefaultValue }
+						className={ styles.button }
+						variant="link"
+					>
+						{ __( 'Load default strings', 'jetpack-boost' ) }
```

Comment:
I agree. The values aren't just strings. They are handles. I'd suggest calling them "default handles" as well.

Reply:
I don't mind calling them handles, I'm worried that a lot of users have no idea what that means. We're mentioning strings everywhere.

Reply:
Changed in [`34fde9b`](https://github.com/Automattic/jetpack/pull/40496/commits/34fde9bb9eedd7252c16fc4a3e5593b4d04174a2).

We should probably change the labels as well.

Reply:
I should have looked more carefully at the modules.

In the JS module:
1. "Scripts" is used to describe the files loaded. 
2. The toggle says, "Exclude JS Strings".
3. The sub header says, "Exclude JS Strings".

In the CSS module:
1. "Styles" is used to describe the files loaded.
2. The toggle says, "Exclude CSS Strings".
3. The sub header says, "Exclude CSS Strings".

This patch adds a button saying, "Load default handles".

Would it be better to use "scripts" in the JS module, and "styles" in the CSS module? This is what it might look like:

![Screenshot 2024-12-11 at 12 06 48](https://github.com/user-attachments/assets/e19b0438-8038-402d-b6c4-2679f717faad)

Reply:
What if we drop "Scripts/Styles" from the label and just say "Exclude JS" or "Exclude CSS".

Then we could update the description to say "handles" and maybe add a link to a support article that mentions how to identify those? Then the button will be "Load default handles".

Reply:
That makes sense when I say "JS scripts" out loud, as in "JavaScript scripts". It's a bit cumbersome.
I'd go one step further and change that last string to "Load Defaults" too. No need to confuse the user with new jargon.

Reply:
> I'd go one step further and change that last string to "Load Defaults" too.

Funny you should say that. I intentionally added what is being loaded (hence the `strings` part), since @haqadn pointed out that it's not clear what "Load defaults" loads when using the button when I initially added it to the dashboard in Cornerstone Pages.

Now I'm confused 😅 

I also think we're getting a bit carried away with details.

Reply:
I think **handle** is fine considering this terminology is coming WP itself, [Example](https://developer.wordpress.org/reference/functions/wp_enqueue_script/).

Reply:
I've updated the UI accordingly in 04a0dfd.